src/lib/annotations/annotatorUtil.js

```diff
@@ -388,6 +388,23 @@ export function eventToLocationHandler(locationFunction, callback) {
     };
 }
 
+/**
+ * Create a JSON object containing x/y coordinates and optionally dimensional information
+ *
+ * @param {number} x - The x position of the location object
+ * @param {number} y - The y position of the location object
+ * @param {Object} [dimensions] - The dimensional information of the location object
+ * @return {Object} - A location object with x/y position information as well as provided dimensional information
+ */
+export function createLocation(x, y, dimensions) {
```

Comment:
I'm not sure we need a separate method for this? Is there a strong reason for breaking this out into a method?